src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -977,12 +977,19 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                     return null;
                 }
 
+                var sourceText = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+
+                var pastedTextSpanOpt =
+                    provider._pasteTrackingService.TryGetPastedTextSpan(sourceText.Container, out var pastedTextSpan)
+                    ? (TextSpan?)pastedTextSpan
+                    : null;
```

Comment:
```suggestion
                        ? (TextSpan?)pastedTextSpan
                        : null;
```

Reply:
I used the formatting from the indenter, which is also the formatting most compatible with combining conditional expressions (i.e. intentional adherence to default behavior).

Reply:
the indenter doesn't indent for clarity.   :)  Another approach that would be clearer here would be: